app/forms/search_keywords_form.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+class SearchKeywordsForm < ApplicationForm
+  attr_reader :keywords, :search_params, :pagination_params
+
+  # rubocop:disable Lint/MissingSuper
+  def initialize(keywords, search_params, pagination_params)
+    @keywords = keywords
+    @search_params = search_params
+    @pagination_params = pagination_params
+  end
+  # rubocop:enable Lint/MissingSuper
+
+  def search_keywords
+    pagination_params[:page] ||= 1
+    pagination_params[:items] ||= 10
```

Comment:
Just for your information, we can handle the pagination params like this.  https://github.com/nimblehq/ewa-payroll-web/blob/develop/app/controllers/concerns/paginable.rb